website/docs/examples/file_reader_gps/README.md

```diff
@@ -0,0 +1,135 @@
+---
+id: file-reader-gps
+title: File Reader GPS Example
+---
+
+This [file_reader_gps_example] (Link to gps_event_reader/main.py) provides a utility for reading and
+parsing GPS messages from the event file. It can process two types of GPS messages: relposned and pvt.
+The user specifies the type of GPS message to parse, and the script reads the corresponding data,
+unpacks it, and prints it to the console.
+
+:::info
+If you haven't already cloned the `farm-ng-amiga` repository, do
+so [**here**](/docs/brain/brain-install.md#clone-the-repository).
+:::
+
+:::tip
+If you already have the log file you want to run with this
+example you can skip to [**set up**](#setup)
+:::
+
+## Download the log file
+
+Now you are going to download the log file that you will use in
+this example.
+[**Click here to download**](https://farm-ng-dev-auto-plot-mvp.s3.us-west-2.amazonaws.com/datasets/western-growers-2022-12-05/events_12052022115852.bin)
```

Comment:
Either give an outdated warning (as in https://github.com/farm-ng/amiga-dev-kit/pull/194) or just link to the log and access data page